api/tests/unit/domain/usecases/account-recovery/get-user-by-account-recovery-demand_test.js

```diff
@@ -0,0 +1,77 @@
+const { catchErr, domainBuilder, expect, sinon } = require('../../../../test-helper');
```

Comment:
Suggestion : Comme tu as créé un dossier account-recovery pour ce test, ça mériterait de déplacer l'autre test `send-email-for-account-recovery_test.js` dans ce même dossier :)